spec/unit/installer/sandbox_dir_cleaner_spec.rb

```diff
@@ -26,18 +27,18 @@ module Pod
       @cleaner.clean!
     end
 
-    it 'does not remove pod or aggregate target support directories' do
+    it 'does not remove pod or aggregate support files and directories' do
       @cleaner.expects(:remove_dir).never
       @cleaner.clean!
     end
 
     it 'cleans up stale headers and keeps pod target headers' do
-      random_pod_private_headers = @pod_target.build_headers.root.join('RandomPod')
+      random_pod_private_headers = @banana_pod_target.build_headers.root.join('RandomPod')
       random_pod_public_headers = @sandbox.public_headers.root.join('PublicPod')
       banana_lib_public_headers = @sandbox.public_headers.root.join('BananaLib')
-      banana_lib_private_headers = @pod_target.build_headers.root.join('BananaLib')
+      banana_lib_private_headers = @banana_pod_target.build_headers.root.join('BananaLib')
       coconut_lib_public_headers = @sandbox.public_headers.root.join('CoconutLibModule')
-      coconut_lib_private_headers = @pod_target.build_headers.root.join('CoconutLibModule')
+      coconut_lib_private_headers = @banana_pod_target.build_headers.root.join('CoconutLibModule')
```

Comment:
@sebastianv1 this appears to had been wrong, did you mean to have this as `@other_pod_target` (and in my PR `@coconut_pod_target`)?

Reply:
I guess it didnt matter